inc/az_ulib_descriptor_api.h

```diff
@@ -229,15 +230,17 @@ typedef void* az_ulib_ipc_interface_handle;
  *                                  the future.
  * @return The #az_ulib_capability_descriptor with the command async.
  */
-#define AZ_ULIB_DESCRIPTOR_ADD_COMMAND_ASYNC(                               \
-    command_name, command_concrete, command_span_wrapper, cancel_concrete)  \
-  {                                                                         \
-    ._internal                                                              \
-        = {.name = AZ_SPAN_LITERAL_FROM_STR(command_name),                  \
-           .capability_ptr_1 = { .command_async = command_concrete },       \
-           .capability_ptr_2 = { .cancel = cancel_concrete },               \
-           .span_wrapper_ptr_1 = { .command_async = command_span_wrapper }, \
-           .flags = (uint8_t)(AZ_ULIB_CAPABILITY_TYPE_COMMAND_ASYNC) }      \
+#define AZ_ULIB_DESCRIPTOR_ADD_COMMAND_ASYNC(                                                     \
+    command_name, command_concrete, command_span_wrapper, cancel_concrete)                        \
+  {                                                                                               \
+    ._internal = {                                                                                \
```

Comment:
nit: align braces on same line like in AZ_ULIB_DESCRIPTOR_ADD_COMMAND above, or give them separate lines in the two above

Reply:
This alignend is defined by the clang-format as required by our code guidelines.

Reply:
so it has to be different within the same file? see line 171 and 200 compared to 236/237

Reply:
Yep, clang-format decides the "best" way. I've just sent you an email how to enable it in VSCode.